api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralQueries.java

```diff
@@ -16,9 +16,9 @@ public static String getPatientsWhoReceivedOneViralLoadResult(
             + "ON p.patient_id = e.patient_id JOIN obs o "
             + "ON o.encounter_id = e.encounter_id "
             + "WHERE p.voided = 0 AND e.voided = 0 AND o.voided = 0 AND "
-            + "e.encounter_type = ${adultoSeguimentoEncounterType} AND (o.concept_id = ${viralLoadConcept} AND o.value_numeric IS NOT NULL) "
-            + "OR (o.concept_id = ${viralLoadQualitativeConcept} AND o.value_numeric IS NOT NULL) "
-            + "AND e.encounter_datetime <= :onOrBefore group by patient_id";
+            + "e.encounter_type = ${adultoSeguimentoEncounterType} AND ((o.concept_id = ${viralLoadConcept} AND o.value_numeric IS NOT NULL) "
+            + "OR (o.concept_id = ${viralLoadQualitativeConcept} AND o.value_coded IS NOT NULL)) "
+            + "AND e.encounter_datetime <= :onOrBefore ";
```

Comment:
@simaomazive can you please move this to 2 queries using union, we have seen this bringing problems in the past

Reply:
This produce the same results as using unions, please consider just putting back the group  by but that will not have any effect... Will just merge it